src/test/java/org/apache/commons/lang3/text/ExtendedMessageFormatTest.java

```diff
@@ -415,9 +415,9 @@ public void testExtendedAndBuiltInFormats() {
         for (final Locale locale : testLocales) {
             final MessageFormat builtins = createMessageFormat(builtinsPattern, locale);
             final String expectedPattern = extendedPattern + builtins.toPattern();
-            DateFormat df = null;
-            NumberFormat nf = null;
-            ExtendedMessageFormat emf = null;
+            DateFormat df;
+            NumberFormat nf;
+            ExtendedMessageFormat emf;
             if (locale == null) {
```

Comment:
locale really shouldn't be null here. If it is, that should be a separate test method.

Reply:
The `testLocales` we're looping through contains the null locale scenario (last value added). So I split this test into the *default locale* scenario and the *available locales* group.